lib/schemas/src/tasks.ts

```diff
@@ -1,9 +1,30 @@
-import { z } from 'zod'
-
 import { type InferSchemaOptions } from './infer'
+import { BabelSchema } from './tasks/babel'
+import { ESLintSchema } from './tasks/eslint'
+import { JestSchema } from './tasks/jest'
+import { MochaSchema } from './tasks/mocha'
+import { NodeSchema } from './tasks/node'
+import { NodemonSchema } from './tasks/nodemon'
+import { Pa11ySchema } from './tasks/pa11y'
+import { PrettierSchema } from './tasks/prettier'
+import { TypeScriptSchema } from './tasks/typescript'
+import { UploadAssetsToS3Schema } from './tasks/upload-assets-to-s3'
+import { WebpackSchema } from './tasks/webpack'
+import { SmokeTestSchema } from './tasks/n-test'
 
 export const TaskSchemas = {
-  'fake schema': z.never()
+  Babel: BabelSchema,
+  Eslint: ESLintSchema,
+  Jest: JestSchema,
+  Mocha: MochaSchema,
+  Node: NodeSchema,
+  Nodemon: NodemonSchema,
+  Pa11y: Pa11ySchema,
+  Prettier: PrettierSchema,
+  TypeScript: TypeScriptSchema,
+  UploadAssetsToS3Schema: UploadAssetsToS3Schema,
+  WebpackSchema: WebpackSchema,
+  SmokeTestSchema: SmokeTestSchema
```

Comment:
**issue**: @cerdfied the keys here need to exactly match the names of the tasks that the options are for. we should really have some kind of type checking or something for that, not sure how. sorry this wasn't obvious!

```suggestion
  UploadAssetsToS3: UploadAssetsToS3Schema,
  Webpack: WebpackSchema,
  NTest: SmokeTestSchema
```

Reply:
i've rebased and amended your original commits to fix this